### PR TITLE
Hugging Face Labs PR

### PR DESCRIPTION
**Pull request by jbdel**

### CODE REVIEW

**Comment by jbdel** [comment]
**Hugging Face Toggle (Code, Data and Media)**
Rreference is LABS-142. 

If the arxiv_id exist on the Hugging Face paper page, fetch the paper page and resources (dataset, models, spaces) associated to this paper:

hf-labs

Otherwise, suggest to index the paper on the Hugging Face paper page:

hf_nopaper

Thanks,
JB

File: browse/static/js/huggingface.js [hunk 1]
```diff
@@ -0,0 +1,326 @@
...
+
+    // Get the arXiv paper ID from the URL, e.g. "2103.17249"
+    // (this can be overridden for testing by passing a override_paper_id query parameter in the URL)
+    const params = new URLSearchParams(document.location.search);
+    const arxivPaperId = params.get("override_paper_id") || window.location.pathname.split('/').reverse()[0];
+    if (!arxivPaperId) return;
+
+    const huggingfaceApiHost = "https://huggingface.co/api";
+    const huggingfaceRepo = `${huggingfaceApiHost}/arxiv/${arxivPaperId}/repos`;
+    const huggingfacePaperApi = `${huggingfaceApiHost}/papers/${arxivPaperId}?field=comments`;
```
Comment (osanseviero) [inline browse/static/js/huggingface.js:26]: Are comments needed?

```suggestion
    const huggingfacePaperApi = `${huggingfaceApiHost}/papers/${arxivPaperId}`;
```

File: browse/static/js/huggingface.js [hunk 2]
```diff
@@ -0,0 +1,326 @@
...
+
+    const MAX_TAGS = 5; // You can adjust this value as needed
+
+    // Search the HF API for demos that cite this paper
```
Comment (osanseviero) [inline browse/static/js/huggingface.js:30]: This comment should be updated

Comment (jbdel) [reply browse/static/js/huggingface.js:30]: Will do

**Review by jbdel — COMMENTED** [review]
indeed, field param not needed, thanks.

**Review by cbf66 — APPROVED** [review]

**Pull request merged**